screenpy/director.py

```diff
@@ -26,10 +25,10 @@ def __new__(cls: Type[SelfDirector]) -> SelfDirector:
             cls._instance.notebook = {}
         return cls._instance
 
-    def notes(self: SelfDirector, key: str, value: T) -> None:
+    def notes(self: SelfDirector, key: str, value: Any) -> None:
         """Note down a value under the given key."""
         self.notebook[key] = value
 
-    def looks_up(self: SelfDirector, key: str) -> T:
+    def looks_up(self: SelfDirector, key: str) -> Any:
```

Comment:
`mypy` was complaining about `TypeVar` here, saying that if a function is returning a `TypeVar`, it should be accepting the same `TypeVar` in its arguments. It looks like we can't "store" the type in the dictionary like we were trying to, here.